src/components/dev-hub/learn-tabs.js

```diff
@@ -0,0 +1,82 @@
+import React, { useCallback, useEffect, useState } from 'react';
+import styled from '@emotion/styled';
+import { Tabs as LeafyTabs, Tab } from '@leafygreen-ui/tabs';
+import Button from './button';
+
+const StyledTabButton = styled(Button)`
+    border-radius: 0;
+    :active,
+    :hover,
+    :focus {
+        color: ${({ theme }) => theme.colorMap.darkGreen};
+        :after {
+            background-color: ${({ theme }) => theme.colorMap.darkGreen};
+        }
+    }
+`;
+
+const StyledTabs = styled(LeafyTabs)`
+    margin: 0 32px;
```

Comment:
Awesome! Last nitpick is can we use the `large` size option from `src/components/dev-hub/theme.js` in here? [Here is is on GitHub](https://github.com/mongodb/devhub/blob/staging/src/components/dev-hub/theme.js#L47)

No worries! definitely will take a bit of adjustment but for a first PR this is fantastic so please recognize that! Personally, I get very nitpicky with the PRs